Error when Mapping type in (non-local) storage is assigned to (#8977)
ethereum/solidity: Changelog.md

Bugfixes:
 * Optimizer: Fixed a bug in BlockDeDuplicator.

>  * Type Checker: Error message when Mapping type in (non-local) storage is assigned to.

Reviewer: ```suggestion
 * Type Checker: Disallow assignments to storage variables of type ``mapping``.
```
